resourcecontrollerv2/resource_controller_v2_integration_test.go

```diff
@@ -225,6 +225,7 @@ var _ = Describe("Resource Controller - Integration Tests", func() {
 			fmt.Fprintf(GinkgoWriter, "GetResourceInstance() result:\n%s\n", common.ToJSON(result))
 
 			Expect(*result.ID).To(Equal(testInstanceCRN))
+			Expect(result.OnetimeCredentials).ToNot(BeNil())
```

Comment:
`result.OnetimeCredentials` is a pointer so it must be dereferenced before checking its value.
```suggestion
			Expect(*result.OnetimeCredentials).ToNot(BeNil())
```

Reply:
Addressed